public/js/livesearch.js

```diff
@@ -0,0 +1,104 @@
+	//init vars
```

Comment:
Add comment header to file (<https://github.com/Capping-CPCA/Web-App/wiki/Adding-New-Pages#comment-header>)